analysis/import/SonarImporter/src/test/java/de/maibornwolff/codecharta/importer/sonar/dataaccess/SonarMeasuresAPIDatasourceIntegrationTest.java

```diff
@@ -157,4 +154,23 @@ public void createMeasureAPIRequestURI_illegal_character_in_metrics_should_throw
 
         // then throw
     }
+
+    @Test
+    public void getComponents_from_server_if_no_authentication_needed() throws Exception {
```

Comment:
test method convention not consistent. Other test methods use Should and When in the method name. Also now we use underscores again.